src/bin/tikv-server.rs

```diff
@@ -765,6 +791,34 @@ fn get_store_path(matches: &Matches, config: &toml::Value) -> String {
     format!("{}", absolute_path.display())
 }
 
+fn get_server_tags(matches: &Matches, config: &toml::Value) -> Vec<String> {
+    let tags = get_array_value("tags",
+                               "server.tags",
+                               matches,
+                               config,
+                               Some(Vec::new()),
+                               |v| {
+        v.as_slice().and_then(|s| {
+            let mut tags = Vec::new();
+            for x in s {
+                if let Some(t) = x.as_str() {
+                    tags.push(t.to_owned());
```

Comment:
No, that only process tags in config, we still need to process tags in matches below.